webpack.dev.js

```diff
@@ -7,7 +7,7 @@ module.exports = env => merge(common(env), {
   devServer: {
     contentBase: path.join(__dirname, 'dist/'),
     port: 3000,
-    publicPath: 'http://localhost:3000/',
+    publicPath: 'http://localhost:3000/dist/',
```

Comment:
Not sure why you changed this?

Reply:
good point, forgot to revert that after we talked about why the project is structured the way it is!